en/products/vpn/more/ip-address.ftl

```diff
@@ -0,0 +1,51 @@
+
+# Used as an accessible text alternative for an image
+vpn-ip-address-an-example-of-an-ip = An example of an IP address that is a sequence of four numbers, separated by decimals: 123.45.67.89
+vpn-ip-address-if-you-look-up-your = If you look up your IP address, it will look something like this.
+vpn-ip-address-does-it-travel = Does it travel with you?
+
+# Variables
+#   $url (url) - https://www.mozilla.org/products/vpn/more/when-to-use-a-vpn/
+vpn-ip-address-no-your-ip = No. Your IP address is only associated with one location unless you are <a href="{ $url }">using a VPN</a> (we will get more into that later). When you are at your home and connecting to the internet you pay for, you are using one. However, if you check your email at home in the morning, then scan the news at a local coffee shop while waiting for your coffee, and then work from an office, you will have used different IP addresses at each location.
+vpn-ip-address-does-your-ip = Does your IP address change?
+vpn-ip-address-yes-even-if = Yes. Even if you are only using the internet at home, the IP address for your home can change. You can contact your internet service provider (ISP) to change it, but even something as routine as restarting your modem or router because of internet connection problems could result in a change.
+vpn-ip-address-can-more-than = Can more than one device have the same IP address?
+vpn-ip-address-this-is-a = This is a bit of a tricky question — the answer is both yes and no. More than one device can share the same external (public) IP address, but each device will have its own local (private) IP address. For example, your ISP (internet service provider) sets your home up with one external IP address. Since your router is what actually connects to the internet, the IP address is assigned to your router. Your router then assigns a local IP address to each device that is connected to the internet at a time. The external IP address is what is shared with the outside world. Your local IP address is not shared outside of your private home network.
+vpn-ip-address-can-we-run = Can we run out of them?
+
+# Variables
+#   $sr (url) - https://www.siliconrepublic.com/comms/ip-addresses-running-out
+#   $variety (url) - https://variety.com/2019/digital/news/u-s-households-have-an-average-of-11-connected-devices-and-5g-should-push-that-even-higher-1203431225/
+vpn-ip-address-when-the-internet = When the Internet was first designed it used ‘version 4’ addresses. These are 32 bits, which means that we could have up to <a href="{ $sr }">4.2bn addresses</a>. This seemed like enough at the time, but is nowhere near enough in a world where the average U.S. household had <a href="{ $variety }">11 connected devices</a>.
```

Comment:
- Add a comma after `When the Internet was first designed`
- Should we use double quote for `‘version 4’` to be consistent with `“tunnel”` referenced a few times in the VPN pages?